core/src/main/java/org/elasticsearch/index/search/MatchQuery.java

```diff
@@ -355,6 +355,9 @@ protected Query blendTermQuery(Term term, MappedFieldType fieldType) {
                     if (query instanceof FuzzyQuery) {
                         QueryParsers.setRewriteMethod((FuzzyQuery) query, fuzzyRewriteMethod);
                     }
+                    if (query == null) {
```

Comment:
As I said, I didn't check all implementations of `fuzzyQuery()` (there a quiet a few). My guess is you're right, but since the method doesn't have some clear javadocs that state that, I thought it better to check again here, but I can also remove this.